frame/democracy/src/lib.rs

```diff
@@ -1258,9 +1172,9 @@ decl_module! {
 		///   Weight is calculated for the maximum number of vote.
 		/// - Db reads: `ReferendumInfoOf`, `VotingOf`
 		/// - Db writes: `ReferendumInfoOf`, `VotingOf`
-		/// - Base Weight: 19.15 + .372 * R
 		/// # </weight>
-		#[weight = 19_000_000 + 370_000 * Weight::from(T::MaxVotes::get()) + T::DbWeight::get().reads_writes(2, 2)]
+		#[weight = T::WeightInfo::remove_other_vote(T::MaxVotes::get())
+			.max(T::WeightInfo::remove_vote(T::MaxVotes::get()))]
```

Comment:
Why is `remove_vote` weight here?

Reply:
basically if target == origin then it allows to remove a vote from an ongoing referendum (which is what remove_vote does), this means 2 storage write, if target != origin then it doesn't allow it thus the worst case is 1 storage write.

maybe I can change benchmark remove_other_vote to actually be target==origin (which is the worst case)

Reply:
so now benchmark remove_other_vote has been renamed remove_other_vote_different.
Because remove_other_vote_same == remove_vote so no need to have yet another benchmark.

For clarity though I'm ok to introduce remove_other_vote_same